core/templates/services/rte-helper.service.ts

```diff
@@ -98,8 +98,6 @@ angular.module('oppia').factory('RteHelperService', [
           // this service is migrated to Angular and NgbModal is used instead of
           // uibModal. Currently, these custom classes are used for correctly
           // stacking AngularJS modals on top of Angular modals.
-          windowClass: 'forced-modal-stack-over',
-          backdropClass: 'forced-modal-backdrop-stack-over',
```

Comment:
Can you explain this removal please?

Reply:
@iamprayush    these are css property.

and  changing the z-index not working so, I have removed these property and it work fine.
if i am wrong please let me know and also your  suggestion are welcome as guidance.

Thanks.

Reply:
I understand, but please figure out why removing these is making it work. We need to ensure that removing these has no unintended side-effects.

Reply:
hi prayush my exam had been done and now  i am back to work. and i am agree with your above comment when i look for another place where this component is used i found that  removing these two class work fine in creating the exploration section but not work fine in topic and skill dashboard section. 
 also these  two (window class and backdropClass) are css that are adjusting the z-index.
and changing z-index would not work because z-index is not work on staged modals. 
Can you suggest any logic to correct.?  also i am trying my best.

Thanks

Reply:
@iamprayush PTAL